### ISSUE TITLE
Treated images don't align/float left or right

### ISSUE THREAD

**Issue by slambert** [thec4aa/c4aa-org#189]
To reproduce:
1. Create a long paragraph under an image block
2. Select thumbnail and align right for the image 
3. preview

**Comment by laras126** [comment]
Example: http://backtobasics.c4aa.org/2020/11/testing-a-bug/

**Comment by laras126** [comment]
Noting that the not aligning seems to be caused by 
```
[class*="is-style-c4aa-duotone"] {
    display: inline-block;
}
```
And
```
[class*="is-style-c4aa-duotone"], .a-filter-child-img {
    overflow: hidden;
}
```

**Comment by laras126** [comment]
Resolved with #232
